### PR TITLE
Changes based on Warner Robins class

### PR DESCRIPTION
**Pull request by frank-at-adacore**
Typos, reformatting, adding notes, re-order some slide

### CODE REVIEW

File: courses/fundamentals_of_ada/010_overview.rst [hunk 1]
```diff
@@ -707,3 +708,25 @@ Canonical First Program
 
    - :command:`say_hello` (Windows)
    - :command:`./say_hello` (Linux/Unix)
+
+--------------------------------
+"Hello World" Lab - GNATstudio
```
Comment (leogermond) [inline courses/fundamentals_of_ada/010_overview.rst:713]: Aren't labs supposed to go into their own file ?

File: courses/fundamentals_of_ada/020_declarations.rst [hunk 2]
```diff
@@ -261,14 +280,12 @@ Decimal Numeric Literals
 Based Numeric Literals
 ------------------------
 
-* Syntax
-
-   .. code:: Ada
+.. code:: Ada
```
Comment (leogermond) [inline courses/fundamentals_of_ada/020_declarations.rst:283]: pygments supports a BNF lexer: https://pygments.org/docs/lexers/#pygments.lexers.grammar_notation.BnfLexer, we should probably declare such code as bnf

File: courses/fundamentals_of_ada/060_record_types.rst [hunk 3]
```diff
@@ -8,20 +8,36 @@ Introduction
...
 
 ---------------------
-Syntax (Simplified)
+Syntax and Examples
 ---------------------
 
-.. code:: Ada
+* Syntax (simplified)
+
+   .. code:: Ada
```
Comment (leogermond) [inline courses/fundamentals_of_ada/060_record_types.rst:16]: see previous BNF remark

File: courses/fundamentals_of_ada/160_genericity.rst [hunk 4]
```diff
@@ -304,7 +307,7 @@ Generic Subprogram Parameters Defaults
       procedure P;
       procedure Callback_1;
```
Comment (leogermond) [inline courses/fundamentals_of_ada/160_genericity.rst:308]: Should probably be renamed to Callback1, unless underscores are ignored for generic `<>` defaults ?

File: courses/fundamentals_of_ada/180_polymorphism.rst [hunk 5]
```diff
@@ -98,12 +98,10 @@ Testing the type of an object
    TC2 : T'Class := D_Obj;  -- TC2'Tag = D'Tag
```
Comment (leogermond) [inline courses/fundamentals_of_ada/180_polymorphism.rst:98]: The naming is hard to follow, using Root and Child may be easier ?

**Review by leogermond — APPROVED** [review]
Approving, with nitpicks comments

File: courses/fundamentals_of_ada/010_overview.rst (hunk 1 above)
Comment (frank-at-adacore) [reply courses/fundamentals_of_ada/010_overview.rst:713]: This is more a "Setup Check" than an actual lab, which is why I embed it in the lecture. Generally, this is done together to make sure everyone is at the same point before going on

File: courses/fundamentals_of_ada/020_declarations.rst (hunk 2 above)
Comment (frank-at-adacore) [reply courses/fundamentals_of_ada/020_declarations.rst:283]: Currently, everything we put into the RST is understood by Pandoc. If we can get BNF to work "for free" (no installs needed) then I would consider it. However, using "Ada" means all of the keywords are highlighted, which (to me) is a benefit

File: courses/fundamentals_of_ada/160_genericity.rst (hunk 4 above)
Comment (frank-at-adacore) [reply courses/fundamentals_of_ada/160_genericity.rst:308]: Right - I updated the comment, but forgot to update the code

File: courses/fundamentals_of_ada/180_polymorphism.rst (hunk 5 above)
Comment (frank-at-adacore) [reply courses/fundamentals_of_ada/180_polymorphism.rst:98]: Done (I prefer parent and child :)

File: courses/fundamentals_of_ada/020_declarations.rst (hunk 2 above)
Comment (leogermond) [reply courses/fundamentals_of_ada/020_declarations.rst:283]: After playing with the BNF syntax color for a bit I agree. First of the BNF is hardly even colored with it, then using the Ada coloration we keep it somewhat consistent between examples, which is a great plus.

File: courses/fundamentals_of_ada/180_polymorphism.rst [hunk 6]
```diff
@@ -90,20 +90,21 @@ Testing the type of an object
...
-   B2 : Boolean := (TC1 in D'Class); -- False
-   B3 : Boolean := (DC in T'Class);  -- True
-   B4 : Boolean := (DC in D'Class);  -- True
+   type Parent is tagged null record;
+   type Child is new Parent with null record;
+   Parent_Obj : Parent; -- Parent_Obj'Tag = Parent_T'Tag
+   Child_Obj  : Child;  -- Child_Obj'Tag = Child'Tag
+   Parent_Class_1 : Parent'Class := Parent_Obj;
+                    -- Parent_Class_1'Tag = Parent'Tag
+   Parent_Class_2 : Parent'Class := D_Obj;
```
Comment (leogermond) [inline courses/fundamentals_of_ada/180_polymorphism.rst:99]: D_Obj should be Child_Obj

File: courses/fundamentals_of_ada/180_polymorphism.rst [hunk 7]
```diff
@@ -90,20 +90,21 @@ Testing the type of an object
...
+                    -- Parent_Class_2'Tag = Child'Tag
+   Child_Class    : Child'Class := Child(Parent_Class_2);
+                    -- Child_Class'Tag  = Child'Tag
+
+   B1 : Boolean := Parent_Class_1 in Parent'Class;      -- True
+   B2 : Boolean := Parent_Class_1'tag = D'Class'tag;    -- False
```
Comment (leogermond) [inline courses/fundamentals_of_ada/180_polymorphism.rst:105]: D'Class'Tag shoudl be Child'Class'Tag

**Review by leogermond — DISMISSED** [review]
Two minor code renaming errors

**Review by leogermond — APPROVED** [review]
